{{cookiecutter.repostory_name}}/bin/rotate-files.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
```

Comment:
Kind of dangerous script. I would prefer to have `rotate-backups.py` that works only in `.backups` without the possibility to "rotate" something we don't know

Reply:
Done